comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -22,10 +22,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-const (
-	// GlobalEntityID defines the entity ID that holds global tags
-	GlobalEntityID = "internal://global-entity-id"
+// GlobalEntityID defines the entity ID that holds global tags
+var GlobalEntityID = types.NewEntityID("internal", "global-entity-id")
```

Comment:
Since now it not anymore a `const` 
can we make `GlobalEntityID` it private? and provide a function to access it. 
The main reason is that is should be possible for a user (another package) to modify it.

Reply:
You are right 👍 

I updated it, thanks for pointing this out.